test_communication/test/test_messages_c.cpp

```diff
@@ -1101,68 +1102,37 @@ void get_message(test_msgs__msg__BoundedSequences * msg, size_t msg_num)
   }
 }
 
+#define EXPECT_SEQUENCE_EQ(pairwise_check, a, b) \
+  do { \
+    EXPECT_EQ(a.size, b.size) << "Sequences have different length"; \
+    for (size_t i = 0; i < std::min(a.size, b.size); ++i) { \
+      pairwise_check(a.data[i], b.data[i]) << "Sequences differ at element " << i; \
+    } \
+  } while (0)
+
+#define EXPECT_RCSTR_EQ(a, b) EXPECT_EQ(std::string(a.data, a.size), std::string(b.data, b.size))
```

Comment:
@rotu what does `RCSTR` stands for? `rcutils` C string? Consider not using an abbreviation.

Reply:
ROS IDL-generated C STRING. What would you prefer it be called?

Reply:
renamed